code/mission/missionload.cpp

```diff
@@ -76,6 +78,20 @@ void ml_update_recent_missions(char *filename)
 	Assert(Num_recent_missions <= MAX_RECENT_MISSIONS);
 }
 
+bool mission_is_ignored(const char *filename)
+{
+	SCP_string filename_no_ext = filename;
+	drop_extension(filename_no_ext);
+
+	for (auto &ii: Ignored_missions) {
+		if (ii == filename_no_ext) {
```

Comment:
I think it would make more sense if this was a case insensitive comparison since the FSO file system is case insensitive as well.

Reply:
Ok.